### DISCUSSION TITLE
Stack problem processing TerraSAR-X images with stackStripMap.py

### DISCUSSION THREAD

**Discussion by fgdiazm** [isce-framework/isce2#615] [Q&A]
Good day everyone

I was finally able to process Terrasar-X using stackStripMap.py, upgrading the ISCE-2 version however when generating the Stack with Mintpy.

The problem now is when I generate the Stack in Mintpy and apply the PYAPS corrections, which is reflected in the final result "geo_velocity", as you can see the image is divided into 2.
![image](https://user-images.githubusercontent.com/92444444/200122558-cec49a96-4eba-43a2-85cb-dcffa97eb860.png)

I did the test, this time without applying Pyaps, the result "geo_velocity", no longer looks "split".

![image](https://user-images.githubusercontent.com/92444444/200122704-870762be-8e4b-48a3-b54c-12d932a64a0c.png)

Check the files los.rdr.rsc, lon.rdr.rsc and lat.rdr.rsc, where the angles of incidence do not correspond to the set of images, they should be between 27° and 30° approx.
![image](https://user-images.githubusercontent.com/92444444/200122722-c922b760-c108-444e-b86d-bc639071a9b8.png)

can be seen in geometryRadar.png

![image](https://user-images.githubusercontent.com/92444444/200122726-a1371bed-a85a-484f-b8cb-43961f06dfcc.png)

And timeseries_ERA5_wrap10.png

![image](https://user-images.githubusercontent.com/92444444/200122742-305cda08-c0e6-429d-8508-e07eee723805.png)

This does not happen when I process Sentinel-1 through stackSentinel.py, in Mintpy when applying PYAPS I have no problem
![image](https://user-images.githubusercontent.com/92444444/200122775-c9f73361-b393-4683-9026-80c3f4b8eec3.png)

I appreciate who can help me solving this

Best

Fredy D.

**Comment by fdelgadodelapuente** [comment]
Hi Fredy,

Your coherent area is too small for reliable asessing the quality of the atmospheric correction. If the data is not coherent over a large area of the swath, there is no way to know if the corrections is reducing or decreasing the effect of the phase delays, unless you use external data like GPS. Hence you should not use them to correct interferograms that are severely decorrelated like in your study.  

Your Sentinel-1 time series is not masked and you have to get rid of the SNAPHU_MCF unwrapping errors (areas with high frequency noise outside of the Barranquilla city)

Francisco
